Cubical/Categories/Instances/PosetCat/Base.agda

```diff
@@ -0,0 +1,45 @@
+{-# OPTIONS --safe #-}
+
+module Cubical.Categories.Instances.PosetCat.Base where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Categories.Category
+open import Cubical.Relation.Binary.Poset
+
+open import Cubical.Categories.Instances.PosetCat.Monotone
+open import Cubical.Categories.Instances.PosetCat.MonotoneAdjoint
+
+private
+  variable
+    ℓ ℓ' : Level
+
+open Category
+
+-- Category of Posets
+PosetCat : {ℓ ℓ' : Level} → Category
```

Comment:
1. The universe levels should be explicit arguments
2. The naming convention in the library is to use ALLCAPS for these large categories so this should be `POSET ℓ ℓ'`. E.g. `SET` and `FUNCTOR`

Reply:
Fixed.